### ISSUE TITLE
UI - Remove header title color transition on larger display

### ISSUE THREAD

**Issue by funabab** [kwasu-ng/gdsckwasu-hacktoberfest21#28]
Currently the header text color transition on profile slide change.

While this look good on mobile, sometimes on large display transitioned color might make the header title barely visible.

A simple fix would be to remove color transition on larger display.
This can easily be done by setting the color and text-shadow properties of h1 in this media query body

https://github.com/kwasu-ng/gdsckwasu-hacktoberfest21/blob/da95b52356465e448ed2f253c73634830689839c/src/components/styled/HeaderContainer.styled.jsx#L21

```
text-color: #2b2b2b
text-shadow-color: #818181
```

**Comment by Tamsauce** [comment]
Yes, I can tackle this!  Please assign and I will work on this first thing in the morning!

**Comment by Caleb335** [comment]
Awesome!

Assigned you just now.

**Comment by Tamsauce** [comment]
@Caleb335 sorry for the delay.  I am having issues with the npm run dev.  For some reason it is not working in my environment
.

**Comment by funabab** [comment]
> @Caleb335 sorry for the delay. I am having issues with the npm run dev. For some reason it is not working in my environment .

What issue are you having, can you paste here any errors you are getting?

**Comment by Tamsauce** [comment]
'next' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! hacktoberfest21@0.1.0 dev: `next dev`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the hacktoberfest21@0.1.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Thugh\AppData\Roaming\npm-cache\_logs\2021-10-31T22_36_15_930Z-debug.log

**Comment by funabab** [comment]
have you tried deleting the node_modules folder and rerunning the npm install command?

**Comment by Tamsauce** [comment]
Oh man!!  It is always the easiest solutions that solve the problem!!  Thanks

**Comment by funabab** [comment]
That's a fact. 😀
Glad you have it running now

**Comment by Caleb335** [comment]
Thank you for helping @funabab! 😁
